themes/default/content/docs/intro/cloud-providers/azure/setup.md

```diff
@@ -58,6 +58,22 @@ A Service Principal is an application in Azure Active Directory with three autho
 secret, and a tenant ID.  (These are often simply called `appId`, `password`, and `tenant`, respectively.)  Using a
 Service Principal is the recommended way to connect Pulumi to Azure in a team or CI setting.
 
+## Using certificate for Service Principal
```

Comment:
I think it should be `###`. It would also make sense to put the section down below at the end of the doc, after we described how a service principal is created.
```suggestion
##3 Using a Certificate for a Service Principal
```